docs/koji.md

```diff
@@ -89,9 +89,13 @@ Data which is placed here includes
   integration is enabled
 - `build.extra.image.parent_build_id` (int): Koji build id of the parent image,
   if found
-- `build.extra.image.parent_image_builds` (map of maps): With each parent image
-  given in the Dockerfile as a key, its value being a map representing the Koji
+- `build.extra.image.parent_image_builds` (map of maps): Keys are parent images
+  given in the Dockerfile, which are regular images and
+  not 'scratch' or custom image ('koji/image-build), and includes registry,
+  organization and tag, its value being a map representing the Koji
```

Comment:
Is the content of `build.extra.image.parent_image_builds` the same as it was before? If the content changed, is there any risk that some existing automation would be broken by this?

Reply:
content didn't change but I wanted to be more specific because what it said before didn't take in account scratch or custom images

Reply:
I thought so, just wanted to make sure, thanks